docs/integration/windows-integration.md

```diff
@@ -0,0 +1,75 @@
+title: Sematext Windows Monitoring
+description: Monitor Windows bare metal servers and VMs using a lightweight agent and visualize your full infrastructure in Sematext Cloud
+
+
+## Windows Services
+
+Sematext Monitoring now includes support for Windows Services. Starting from Sematext Agent [version 3.5.0](https://sematext.com/docs/agents/sematext-agent/releasenotes/#version-350), you can collect and monitor the status of any Windows Service available in your system.
+
+### Metrics
+
+The [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/), starting from version 3.5.0, is capable of collecting metrics related with the status of Windows services. At present, the primary metric supported is the status of each service including a number of statistics, which allows users to determine whether a service is running, stopped, or experiencing issues.
+
+
+#### List of available metrics
+
+|Name| Type  | Description |
+|----|-------|-------------|
+|windows.service.status            | number |status of a Windows service|
+|windows.service.total             | gauge  |total number of Windows services|
+|windows.service.total.running     | gauge  |total number of running Windows services|
+|windows.service.total.not_running | gauge  |total number of non running Windows services|
+
+
+#### Available Windows Services statuses 
+
+Below is a list of all the possible values (statuses) of the `windows.service.status` metric.
+
+|Value | Service Status  |
+|------|-----------------|
+|1     | stopped         |
+|2     | start pending   |
+|3     | stop pending    |
+|4     | running         |
+|5     | continue pending|
+|6     | pause pending   |
+|7     | paused          |
+
+#### Available Tags
+
+Below is a list of available [tags](https://sematext.com/docs/tags/) for the `windows.service.status` metric. Tags can be used to retrieve individual service names or process IDs related with a specific service. Edit any chart in the Infrastructure > Windows > Top Services report to understand how you can use a the `windows.service.status`, grouped by a tag.
```

Comment:
> to understand how you can use a the `windows.service.status`, grouped by a tag.

Other than "a the", I'm not really sure what this is telling the reader.... especially this "grouped by a tag". Maybe something needs rewording here.